packages/eas-cli/src/credentials/manager/ManageAndroid.ts

```diff
@@ -163,29 +161,21 @@ export class ManageAndroid {
     );
     const appLookupParams = await getAppLookupParamsFromContextAsync(ctx, gradleContext);
     if (action === AndroidActionType.CreateKeystore) {
-      const selectBuildCredentialsResult = await new SelectAndroidBuildCredentials(
+      const keystore = await new CreateKeystore(appLookupParams.account).runAsync(ctx);
+      const selectBuildCredentialsResult = await new CreateAndroidBuildCredentials().runAsync();
+      await ctx.android.createAndroidAppBuildCredentialsAsync(ctx.graphqlClient, appLookupParams, {
+        ...selectBuildCredentialsResult,
+        androidKeystoreId: keystore.id,
+      });
+    } else if (action === AndroidActionType.SetDefaultKeystore) {
+      const setDefaultBuildCredentialResult = await new SetDefaultAndroidKeystore(
         appLookupParams
       ).runAsync(ctx);
-      const keystore = await new CreateKeystore(appLookupParams.account).runAsync(ctx);
-      if (
-        selectBuildCredentialsResult.resultType ===
-        SelectAndroidBuildCredentialsResultType.CREATE_REQUEST
-      ) {
-        await ctx.android.createAndroidAppBuildCredentialsAsync(
+      if (setDefaultBuildCredentialResult?.androidKeystore) {
+        Log.log('Setting default build credential for this project...');
```

Comment:
```suggestion
        Log.log('Setting the default build credentials for this project...');
```
Also Setting sounds to me a bit wierd, maybe "Changing the default..."/"Switching"/"Updating" ?